src/core/channels/channel.ts

```diff
@@ -55,12 +56,13 @@ export default class Channel extends EventsDispatcher {
     this.subscriptionPending = false;
   }
 
-  /** Handles an event. For internal use only.
+  /** Handles a message. For internal use only.
    *
-   * @param {String} event
-   * @param {*} data
+   * @param {Message} message
    */
-  handleEvent(event : string, data : any) {
+  handleMessage(message: Message) {
+    var event = message.event;
+    var data = message.data;
```

Comment:
Can we use `const` (or `let`) instead of `var`?

Reply:
We could and ultimately should, but I think this is a broader question about adopting modern JS conventions and a more modern transpiling step. Not something we should necessarily start doing now.